docs/dev/annual_updates.rst

```diff
@@ -22,34 +22,23 @@ into integrating the 2020 data.
 
 1. Obtain Fresh Data
```

Comment:
These changes look good to, but we will need to remember to update this again once automated archives are working